snapcraft/plugins/kbuild.py

```diff
@@ -122,6 +122,21 @@ def __init__(self, name, options, project):
         if logger.isEnabledFor(logging.DEBUG):
             self.make_cmd.append('V=1')
 
+    def enable_cross_compilation(self):
+        logger.info('Cross compiling kernel target {!r}'.format(
```

Comment:
Yeah, either drop or move just this bit to the `kernel plugin` so that in that plugin we see

```
def enable_cross_compilation(self):
    logger.info('Cross compiling kernel target {!r}'.format(
        self.project.kernel_arch))
    super().enable_cross_compilation()
```

And no message in this plugin. This is just here as the kernel makefiles take a long time to load and as a user it was satisfactory to see that you were doing the right thing.